src/actions/functions.js

```diff
@@ -14,6 +14,11 @@ import {
 import { generateCategories } from '../utils/generateTemplatesCategories'
 
 const functionsActions = {
+  deleteFunction: (func, project) => dispatch => {
+    return functionsApi.deleteSelectedFunction(func, project).then(result => {
+      console.log(result)
```

Comment:
I think that a better thing to do here than printing to console is to show the use a success notification with the text “Function deleted successfully” :)